controller/system_backup_controller_test.go

```diff
@@ -371,3 +377,35 @@ func fakeSystemRolloutNamespace(c *C, informerFactory informers.SharedInformerFa
 	err = nsIndexer.Add(namespace)
 	c.Assert(err, IsNil)
 }
+
+func fakeSystemRolloutSnapshot(fakeObj *longhorn.Snapshot, c *C, informerFactory lhinformers.SharedInformerFactory, client *lhfake.Clientset) {
```

Comment:
Move this to `system_restore_controller_test.go`.

Reply:
Sure, and moved. Thanks